Cannot find module '@sveltejs/adapter-node'

**Describe the bug**
clean install of the sveltekit demo produces an error when attempting to run

**Logs**
> bastinald@Kevins-MacBook-Pro llc-app % npm run dev
> 
> > ~TODO~@0.0.1 dev
> > svelte-kit dev
> 
> Cannot find module '@sveltejs/adapter-node'
> Require stack:
> - /Users/bastinald/Sites/llc-app/svelte.config.cjs
> Error: Cannot find module '@sveltejs/adapter-node'
> Require stack:
> - /Users/bastinald/Sites/llc-app/svelte.config.cjs
>     at Function.Module._resolveFilename (node:internal/modules/cjs/loader:924:15)
>     at Function.Module._load (node:internal/modules/cjs/loader:769:27)
>     at Module.require (node:internal/modules/cjs/loader:996:19)
>     at require (node:internal/modules/cjs/helpers:92:18)
>     at Object.<anonymous> (/Users/bastinald/Sites/llc-app/svelte.config.cjs:1:14)
>     at Module._compile (node:internal/modules/cjs/loader:1092:14)
>     at Object.Module._extensions..js (node:internal/modules/cjs/loader:1121:10)
>     at Module.load (node:internal/modules/cjs/loader:972:32)
>     at Function.Module._load (node:internal/modules/cjs/loader:813:14)
>     at ModuleWrap.<anonymous> (node:internal/modules/esm/translators:201:29)

**To Reproduce**
```
npm init svelte@next my-app
cd my-app
npm install
npm run dev
```

select the `demo`, not the `skeleton` when installing

**Expected behavior**
the app to build and open in my browser

browser = chrome

--- a/.changeset/thin-planets-obey.md
+++ b/.changeset/thin-planets-obey.md
@@ -1,0 +1,5 @@
+---
+'create-svelte': patch
+---
+
+Fix global.d.ts rename, Windows build issue, missing adapter-node

--- a/packages/create-svelte/scripts/build-templates.js
+++ b/packages/create-svelte/scripts/build-templates.js
@@ -64,10 +64,7 @@ async function generate_templates(shared) {
 			// also needed for JS projects if people turn on "checkJs" in their jsonfig
 			if (file.name.endsWith('.d.ts')) {
 				if (file.name.endsWith('global.d.ts')) js.push(file);
-				continue;
-			}
-
-			if (file.name.endsWith('.ts')) {
+			} else if (file.name.endsWith('.ts')) {
 				const transformed = transform(file.contents, {
 					transforms: ['typescript']
 				});
@@ -161,7 +158,7 @@ async function generate_shared() {
 			let name = file;
 
 			if (file.startsWith('+') || file.startsWith('-')) {
-				const [conditions, ...rest] = file.split('/');
+				const [conditions, ...rest] = file.split(path.sep);
 
 				const pattern = /([+-])([a-z]+)/g;
 				let match;

--- a/packages/create-svelte/templates/default/package.template.json
+++ b/packages/create-svelte/templates/default/package.template.json
@@ -7,6 +7,7 @@
 		"start": "svelte-kit start"
 	},
 	"devDependencies": {
+		"@sveltejs/adapter-node": "workspace:*",
 		"@sveltejs/kit": "workspace:*",
 		"svelte": "^3.29.0",
 		"vite": "^2.1.0"

